marklogic-client-api/src/main/java/com/marklogic/client/expression/FnExpr.java

```diff
@@ -2002,7 +2002,7 @@ public interface FnExpr {
   */
   public ServerExpression timezoneFromTime(ServerExpression arg);
 /**
-  * Returns a sequence of strings constructed by breaking the specified input into substrings separated by the specified pattern. The specified pattern is not returned as part of the returned items.
+  * Returns a sequence of strings contructed by breaking the specified input into substrings separated by the specified pattern. The specified pattern is not returned as part of the returned items.
```

Comment:
Looks like a typo in the file that this was generated from? We should fix this since it's user-facing.